free-tunes/ExploreMusic/Repository/TracksModel.swift

```diff
@@ -22,16 +22,21 @@ struct Tracks: Codable {
 }
 
 struct Track: Codable {
-    let title: String
-    let artist: Artist
-    let album: Album
-    let trackLink: String
+    private let trackTitle: String
+    private let trackArtist: Artist
+    private let trackAlbum: Album
+    private let previewLink: String
     enum CodingKeys: String, CodingKey {
-        case title
-        case artist
-        case album
-        case trackLink = "preview"
+        case trackTitle
+        case trackArtist
+        case trackAlbum
+        case previewLink = "preview"
     }
+    
+    var trackLink: String {get {return previewLink}}
```

Comment:
Then you won't need this 😄

Reply:
Are you going to change this?

Reply:
still waiting to hear from brandon

Reply:
Check above comment 😄 you can remove this it isn't necessary 👍